### ISSUE TITLE
Cypress does not work when installed to a project

### ISSUE THREAD

**Issue by jayporta** [cypress-io/cypress#21049]
### Current behavior

Run `npm install --save-dev cypress`

Run `./node_modules/.bin/cypress open`

Result:

```
It looks like this is your first time using Cypress: 9.5.4


Cypress failed to start.

This may be due to a missing library or dependency. https://on.cypress.io/required-dependencies

Please refer to the error below for more details.

----------

/Users/jay/Library/Caches/Cypress/9.5.4/Cypress.app/Contents/MacOS/Cypress: bad option: --no-sandbox
/Users/jay/Library/Caches/Cypress/9.5.4/Cypress.app/Contents/MacOS/Cypress: bad option: --smoke-test
/Users/jay/Library/Caches/Cypress/9.5.4/Cypress.app/Contents/MacOS/Cypress: bad option: --ping=794

----------

Platform: darwin-x64 (21.4.0)
Cypress Version: 9.5.4
```

### Desired behavior

Clean install should work as expected with `./node_modules/.bin/cypress open`, `$(npm bin)/cypress open`, or `npx cypress open`.

### Test code to reproduce

This is not related to a failing test

### Cypress Version

9.5.4

**Comment by jayporta** [comment]
No. I’ve tried deleting the Cypress directory in my user Library/Caches directory and installing older versions, but they always create the Caches/Cypress directory again and then throw errors that data is missing in that directory. It never seems to read from the project node_modules. 

I’ve temporarily downloaded and installed the Cypress app, but I’m on a team with other devs so this is not a long term solution.

**Comment by dmitriisem** [comment]
Same issue. 
Tried to install older versions but it didn't help
Also yesterday updated MacOS to 12.3.1 v. Mb this is the reason 
`dmitriisemenov@Dmitriis-MBP learncyrpress % npx cypress open
It looks like this is your first time using Cypress: 9.5.4


Cypress failed to start.

This may be due to a missing library or dependency. https://on.cypress.io/required-dependencies

Please refer to the error below for more details.

----------

Command failed with Unknown system error -86: /Users/dmitriisemenov/Library/Caches/Cypress/9.5.4/Cypress.app/Contents/MacOS/Cypress --no-sandbox --smoke-test --ping=524
spawn Unknown system error -86

----------

Platform: darwin-x64 (21.4.0)
Cypress Version: 9.5.4
`

**Comment by dmitriisem** [comment]
The first answer here worked for me:

https://stackoverflow.com/questions/69097958/cypress-io-on-apple-silicon-m1-with-yarn-cypress-failed-to-start-this-may-be

**Comment by jayporta** [comment]
Thanks but installing Rosetta didn’t help

**Comment by roshan521** [comment]
As instructed, cypress does not work after installation I guess we need to configure its config by ourselves look inot this configuration manage same at our project  [https://github.com/cypress-io/cra-template-cypress-typescript/tree/master/template](url)
